comp/haagent/def/role.go

```diff
@@ -3,15 +3,16 @@
 // This product includes software developed at Datadog (https://www.datadoghq.com/).
```

Comment:
Note: weird, git thinks that I renamed `comp/haagent/impl/utils_test.go → comp/haagent/def/role.go` 🤷